alliance/apps/shared/views/index.py

```diff
@@ -8,7 +8,7 @@
 @login_required
 def index(request):
     if request.method == 'POST':
-        form = ChooseTeamForm(request, request.POST)
+        form = ChooseTeamForm(request, request.POST or None)
```

Comment:
This doesn't fully solve the problem in issue #18. It's still possible for someone to post gibberish to this form and get an error because we don't assign to the variable `team` if the request method is POST and the form is invalid.

Passing `request.POST or None` says to pass `request.POST` as long if it's not empty, otherwise pass `None`.